compiler/AST/primitive.cpp

```diff
@@ -679,6 +679,9 @@ initPrimitive() {
   // These are used for task-aware allocation.
   prim_def(PRIM_SIZEOF, "sizeof", returnInfoSizeType);
 
+  // sizeof(array.eltType), since CPtr.chpl:c_sizeof() is "not well defined"
+  prim_def(PRIM_SIZEOF_RAW, "sizeof_raw", returnInfoSizeType);
```

Comment:
Let's call the new one PRIM_SIZEOF_DATA_ELEMENT since it expects the argument is a _ddata, right?